tests/integration/test_logcollector/test_macos/test_macos_format_basic.py

```diff
@@ -45,7 +42,7 @@ def get_connection_configuration():
     return logcollector.DEFAULT_AUTHD_REMOTED_SIMULATOR_CONFIGURATION
 
 
-@pytest.mark.parametrize('macos_message', macos_log_messages)
+@pytest.mark.parametrize('macos_message', macos_log_messages, ids=[x['id'] for x in macos_log_messages])
```

Comment:
`x["id"]` don't use letters for arrays, use descriptive names, for examples: "id_list"

Reply:
Sure, but actually the array is called macos_log_message. In this case the `x` refers to each element of this array. Each element has a map, that's why we use `x['id]`, but yes, I've renamed it to log_message as each element of the array is a log message.

https://github.com/wazuh/wazuh-qa/pull/1478/commits/98d147f6c4b011f968ed64f884fc532d4834d842